__tests__/product/product.query.test.js

```diff
@@ -186,61 +193,51 @@ describe('Product queries', () => {
     const resultProduct = product.data.getProductById;
     expect(resultProduct).toBeDefined();
     expect(resultProduct).toHaveProperty('name', [
-      { __typename: 'Language', value: 'Very Coool Baggy' },
-      { __typename: 'Language', value: 'ДУЖЕ СУПЕРСЬКИЙ Рюкзачечок' },
+      { value: 'Very Coool Baggy' },
```

Comment:
Please, don't hardcode values which you assert, instead use properties of the object that you have used for the creation of the product. It will reduce the required changes if some data of the product is going to be changed

Reply:
solved